docs/configuration.md

```diff
@@ -106,7 +106,15 @@ To configure different ports use following properties
 * `grpc_inference_port`: Inference gRPC API binding port. Default: 7070
 * `grpc_management_port`: management gRPC API binding port. Default: 7071
 
-Here are a couple of examples:
+To configure max connection age (milliseconds)
+
+* `grpc_inference_max_connection_age_ms`: Inference gRPC max connection age. Default: Infinite
+* `grpc_management_max_connection_age_ms`: Management gRPC max connection age. Default: Infinite
+
+To configure max connection age grace (milliseconds)
+
+* `grpc_inference_max_connection_age_grace_ms`: Inference gRPC max connection age grace. Default: Infinite
```

Comment:
Could you elaborate a bit what grace will mean in this context?

Reply:
Link to api doc would be sufficient

Reply:
Linked the API docs in Readme